build.Dockerfile

```diff
@@ -1,11 +1,8 @@
-FROM golang:1.8
+FROM golang:1.9-alpine
```

Comment:
Do we plan to merge this even when Go 1.9 is still beta? I'd be concerned about that.

Reply:
I think a lot of people need the Proxy Protocol Support (included myself)! Any release date in the scope?